Dockerfile

```diff
@@ -12,6 +12,10 @@ RUN yum install -y epel-release && \
     yum install -y https://kojipkgs.fedoraproject.org//packages/python-djblets/0.9.4/3.el7/noarch/python-djblets-0.9.4-3.el7.noarch.rpm && \
     yum clean all
 
+# Postgre client for wait until the database is ready
+RUN yum install -y postgresql && \
```

Comment:
Absolutely! I first put the RUN there to make it more explicit, because is more a requeriment to make the container work in compose fashion that a reviewboard requeriment.